ui/src/app/workflows/components/workflow-filters/workflow-filters.tsx

```diff
@@ -57,13 +90,45 @@ export function WorkflowFilters(props: WorkflowFilterProps) {
         return results;
     }, [props.workflows, props.phaseItems]);
 
+    const handleNameFilterChange = (item: {title: string; id: string}) => {
+        props.setNamePrefix('');
+        props.setNamePattern('');
+        props.setName('');
```

Comment:
I actually wouldn't suggest clearing these out necessarily; I don't think that's necessarily ideal UX.
If I did an exact search and didn't find anything, I might want to change to a prefix or contains search using the same query